ps_checkout.php

```diff
@@ -79,6 +80,11 @@ class Ps_checkout extends PaymentModule
     // the module each time to get the version
     const VERSION = '1.2.6';
 
+    /**
+     * @var \Monolog\Logger
+     */
+    public $logger;
```

Comment:
let's be more safe and use a private property and a getter, that way you should be able to do something like this to use it `$logger = \Module::getInstanceByName('ps_checkout')->getLogger();`